jupyterhub/handlers/base.py

```diff
@@ -1522,7 +1522,7 @@ async def get(self, user_name, user_path):
             url_path_join(self.hub.base_url, "spawn", user.escaped_name, server_name),
             {"next": self.request.uri},
         )
-        self.set_status(503)
+        self.set_status(424)
```

Comment:
👍 I think there are some surrounding comments still referencing 503. Maybe update those to indicate why we use 424, to distinguish a server being unavailable from regular bad links (404) and infrastructure problems (503) since it's been a valuable piece of information to see how many requests are being made to non-running servers, e.g. due to culling servers when users have still open (or reopened) tabs.